pymc_marketing/mmm/fourier.py

```diff
@@ -331,6 +333,63 @@ def nodes(self) -> list[str]:
             f"{func}_{i}" for func in ["sin", "cos"] for i in range(1, self.n_order + 1)
         ]
 
+    def get_default_start_date(
+        self,
+        today: datetime.datetime,
+        start_date: str | datetime.datetime | None = None,
```

Comment:
The two parameters confuse me a bit. Is there any way of doing it with only one? 

Maybe it is the function and parameter name that is confusing

The test using `fourier.get_default_start_date(today, None)` is what was throwing me off

Reply:
Latest push has : 

- `today` removed from params 
- `isinstance(start_date, (str, datetime.datetime))` added with comment for ruff ignore

Hopefully, the above two fixes are OK for you.